### PR TITLE
Expand Go and Java SDK Test Coverage

### PR DESCRIPTION
**Pull request by mrzzy**
**What this PR does / why we need it**:

Expand SDKs Test Coverage:
- Added unit test to test client.go in Go SDK
- Added unit test to test `FeastClient` in Java SDK.
- Expanded unit test for `Client.get_online_features()` in Python SDK to verify the correct request is submitted.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: sdk/python/tests/test_client.py [hunk 1]
```diff
@@ -201,7 +202,7 @@ def test_version(self, mocked_client, mocker):
         [pytest.lazy_fixture("mock_client"), pytest.lazy_fixture("secure_mock_client")],
     )
     def test_get_online_features(self, mocked_client, mocker):
-        ROW_COUNT = 300
+        ROW_COUNT = 1
```
Comment (woop) [inline sdk/python/tests/test_client.py:205]: Why was the count changed here?

File: sdk/go/client_test.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	tt := []struct {
+		name    string
+		req     OnlineFeaturesRequest
+		recieve OnlineFeaturesResponse
+		want    OnlineFeaturesResponse
+		wantErr bool
+		err     error
+	}{
+		{
+			name: "valid",
```
Comment (zhilingc) [inline sdk/go/client_test.go:25]: This name can be more informative if you're testing for more than just a "valid" request

File: sdk/java/src/main/java/com/gojek/feast/FeastClient.java [hunk 3]
```diff
@@ -39,8 +40,8 @@
 
   private static final int CHANNEL_SHUTDOWN_TIMEOUT_SEC = 5;
 
-  private final ManagedChannel channel;
-  private final ServingServiceGrpc.ServingServiceBlockingStub stub;
+  private ManagedChannel channel;
```
Comment (zhilingc) [inline sdk/java/src/main/java/com/gojek/feast/FeastClient.java:43]: I think you should keep these as is (`final`). if you want to repalce the channel and stub with mocks for testing purposes, you might want to make a protected all args constructor instead.

File: sdk/java/src/main/java/com/gojek/feast/FeastClient.java [hunk 4]
```diff
@@ -161,9 +162,9 @@ public GetFeastServingInfoResponse getFeastServingInfo() {
         .collect(Collectors.toList());
   }
 
-  private FeastClient(ManagedChannel channel) {
+  protected FeastClient(ManagedChannel channel) {
```
Comment (zhilingc) [inline sdk/java/src/main/java/com/gojek/feast/FeastClient.java:165]: same here, leave it as private

File: sdk/python/tests/test_client.py (hunk 1 above)
Comment (mrzzy) [reply sdk/python/tests/test_client.py:205]: Corrected.

File: sdk/java/src/main/java/com/gojek/feast/FeastClient.java (hunk 3 above)
Comment (mrzzy) [reply sdk/java/src/main/java/com/gojek/feast/FeastClient.java:43]: I only need to mock `channel`. Mocking `stub` is not possible as its a `final` class.

File: sdk/java/src/main/java/com/gojek/feast/FeastClient.java (hunk 4 above)
Comment (mrzzy) [reply sdk/java/src/main/java/com/gojek/feast/FeastClient.java:165]: ^ Refer to above.

Comment (mrzzy) [reply sdk/java/src/main/java/com/gojek/feast/FeastClient.java:165]: I only need to mock `channel`. Mocking `stub` is not possible as its a `final` class.

File: sdk/java/src/main/java/com/gojek/feast/FeastClient.java (hunk 3 above)
Comment (mrzzy) [reply sdk/java/src/main/java/com/gojek/feast/FeastClient.java]: Corrected.

File: sdk/go/client_test.go (hunk 2 above)
Comment (mrzzy) [reply sdk/go/client_test.go]: Corrected.

File: sdk/go/client.go [hunk 5]
```diff
@@ -61,17 +61,17 @@ func (fc *GrpcClient) GetOnlineFeatures(ctx context.Context, req *OnlineFeatures
 
 	// strip projects from to projects
 	for _, fieldValue := range resp.GetFieldValues() {
-		var stripFields map[string]*types.Value
+		stripFields := make(map[string]*types.Value)
```
[the remark this answers is not in the document]
Comment (mrzzy) [reply sdk/go/client.go]: Corrected.

File: sdk/java/src/main/java/com/gojek/feast/FeastClient.java (hunk 4 above)
Comment (zhilingc) [reply sdk/java/src/main/java/com/gojek/feast/FeastClient.java:165]: okayy

File: sdk/python/tests/test_client.py (hunk 1 above)
Comment (woop) [reply sdk/python/tests/test_client.py:205]: Hmmm, it is still showing as `ROW_COUNT = 1` to me. Is that expected?

Comment (mrzzy) [reply sdk/python/tests/test_client.py]: :disappointed_relieved: Should be reverted.

**Pull request merged**